docs/organizations/managing-security-and-risk.md

```diff
@@ -105,16 +158,23 @@ The following table describes how item statuses map to deadlines:
 
 ## Item priorities and deadlines
 
-The following table defines item priorities and days to fix the underlying Codacy security issue, based on the severity of the issue:
+The following table defines security item priorities and days to fix the associated issue, based on the importance of the issue:
 
-| Security issue severity | Item priority | Days to close |
-|-------------------------|---------------|---------------|
-| Critical                | Critical      | 30            |
-| Medium                  | Medium        | 90            |
-| Minor                   | Low           | 120           |
+| **Security item<br/>priority** | <br/>Days to close | Codacy<br/>security issue severity | Jira<br/>security issue priority |
+|--------------------------------|--------------------|------------------------------------|----------------------------------|
+| **Critical**                   | 30                 | Critical                           | Blocker                          |
+| **High**                       | 60                 | -                                  | High                             |
+| **Medium**                     | 90                 | Medium                             | Medium                           |
+| **Low**                        | 120                | Minor                              | Low and other                    |
 
 ## Data retention
```

Comment:
This section does not currently match the behavior of the product. I'm hiding it while we work on item deletion when deleting repos or uninstalling the Jira integrations.

```suggestion
<!--
## Data retention
```